src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -441,6 +444,27 @@ private List<DataBlock> GetDependentDataBlocks(Guid releaseId, Guid subjectId)
                 .Where(block => block.DataBlockRequest.SubjectId == subjectId)
                 .ToList();
         }
+        private async Task<Either<ActionResult,bool>> CheckAllDatafilesUploadedComplete(Guid releaseId, ReleaseStatus status)
+        {
+            if (status == ReleaseStatus.Approved)
+            {
+                var filters = TableQuery.CombineFilters(
+                    TableQuery.GenerateFilterCondition("PartitionKey", 
+                        QueryComparisons.Equal, releaseId.ToString())
+                    , TableOperators.And, 
+                    TableQuery.GenerateFilterCondition("Status", 
+                        QueryComparisons.NotEqual, IStatus.COMPLETE.ToString()));
+                
+                var query = new TableQuery<DatafileImport>().Where(filters);
+                var cloudTable = _coreTableStorageService.GetCloudTable("imports");
```

Comment:
I think you can use `_coreTableStorageService.GetTableAsync` which already exists rather than adding GetCloudTable.

Reply:
Yep - well spotted - changed.